gradebookng/tool/src/webapp/WEB-INF/tools/sakai.gradebookng.xml

```diff
@@ -12,6 +12,8 @@
        	<category name="sakai.admin" />
        	-->
 
+		<!-- Allow multiple instances of this tool within one site -->
+		<configuration name="allowMultipleInstances" value="true" />
```

Comment:
this seems to contrast with the sakai property?

Reply:
Setting false the property will override this.